tools/Environments/DevHome.Environments/Helpers/DataExtractor.cs

```diff
@@ -43,8 +43,8 @@ public static List<OperationsViewModel> FillDotButtonOperations(ComputeSystem co
         }
 
         // ToDo: Correct the function used
-        // operations.Add(new OperationsViewModel("Pin To Taskbar", "\uE718", computeSystem.DeleteAsync));
-        // operations.Add(new OperationsViewModel("Add to Start Menu", "\uF0DF", computeSystem.DeleteAsync));
+        // operations.Add(new OperationsViewModel("Pin To Taskbar", "\uE718", computeSystem.));
+        // operations.Add(new OperationsViewModel("Add to Start Menu", "\uE8A9", computeSystem.));
         return operations;

```

Comment:
should we remove the comment? Or are you planning on updating this before build?

Reply:
This is in place so that we don't have to search for the Glyphs again. I don't plan on removing this.

Reply:
I meant the top of it has a ToDo statement so I was under the impression that it was resolved. If the todo is still active we should probably make a github issue. Anis will track them that way.